src/http/httpClient/Microsoft.Kiota.Http.HttpClientLibrary.csproj

```diff
@@ -15,11 +15,11 @@
     <!-- suppressed because of this CVE https://github.com/advisories/GHSA-hh2w-p6rv-4g7w 
      The target application is the one which will resolve the correct version
      when the version range is updated to > 8.0.4 in the future, remove the nowarn suppression -->
-    <PackageReference Include="System.Text.Json" Version="[6.0,9.0)" NoWarn="NU1903" />
+    <PackageReference Include="System.Text.Json" Version="[6.0,10.0)" NoWarn="NU1903" />
```

Comment:
```suggestion
    <PackageReference Include="System.Text.Json" Version="6.0" NoWarn="NU1903" />
```

Reply:
fixing to a specific version of STJ is going to create problems for people with different versions in their applications (most people I expect at this point)

Reply:
For more context you can also checkout issues like https://github.com/microsoft/kiota-dotnet/issues/218 and https://github.com/microsoft/kiota-serialization-json-dotnet/issues/208 as well @calebkiage